README.md

```diff
@@ -13,11 +13,37 @@ Actions information and common workflows for Qualcomm Linux repositories.
 
 ## Workflows
 
-Common, ["reusable workflows"](https://docs.github.com/en/actions/sharing-automations/reusing-workflows) should go in the `.github/workflows` directory for this repository.
+**multi-checker** This workflow is a reusable workflow to run a series of preflight checks on your code. The checks include:
 
-### Examples
+* **[Repolinter](https://github.com/qualcomm-linux/qli-actions)**: Checks the repository for consistency and adherence to coding standards.
+* **[Semgrep](https://github.com/qualcomm-linux/qli-actions)**: Runs a static analysis tool to detect potential security vulnerabilities and coding errors.
+* **[Copyright-License-Detector](https://github.com/qualcomm/copyright-license-checker-action)**: Checks for proper copyright and licensing information in the code.
+* **[PR-Check-Emails](https://github.com/qualcomm/commit-emails-check-action)**: Verifies that the commit emails are properly formatted.
 
-TBD
+### Examples
+In order to call multi-checker workflow please add below file to your repository's under`.github/workflows` directory.
```

Comment:
... or enable it in your repo via Actions tab (template). Sample instructions I adapted from what we already had for repolinter:

1. Navigate to repo that requires this workflow
1. Click on Actions
1. If you have existing workflows defined in the repo, click "New workflow", else skip to next step
1. Locate the "By Qualcomm" section and click "Configure" under "QLI Preflight Checker Workflow"
Click "Commit changes...", select "Commit directly to the main branch" (or feel free to create a new branch and start a PR), ensure your QC email is selected under "Commit Email", and then click "Sign off and commit changes"
This will create a GitHub Workflow file in your repo under the path .github/workflows/

Reply:
OK I'll merge for now but let's augment the instructions with how to enable via the UI as well